notebooks/metatranscriptomics_pipeline_iSeqnorm.ipynb

```diff
@@ -0,0 +1,1477 @@
+    "\n",
+    "After sequencing the equal volume pool, a file with the read counts associated with all samples will be imported and used to calculate pooling volumes to generate a sequence count normalized **final pooling pick list**.\n",
+    "\n",
+    "Lastly, the per-sample information from the whole run can be combined to automatically produce **sample sheet** to demultiplex sequence data from Illumina sequencers."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Workflow for Quality Control (QC) of the total nucleic acid extraction\n",
+    "\n",
+    "This portion of the notebook will read in the output of the MiniPico quantification assays (gDNA / RNA)\n",
+    "\n",
+    "As inputs, it requires:\n",
+    "1. A tab-delimited row-wise plate map that indicates the sample name, well location, and blank status of each sample on the 384 well plate.\n",
+    "\n",
+    "You can use this google sheet template to generate your plate map:\n",
+    "https://docs.google.com/spreadsheets/d/1xPjB6iR3brGeG4bm2un4ISSsTDxFw5yME09bKqz0XNk/edit?usp=sharing\n",
+    "\n",
+    "The workflow then:\n",
+    "1. reads in the plate map and constructs a dataframe\n",
+    "2. imports quant files and merged DNA/RNA concentration data. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 1: read in plate map\n",
+    "\n",
+    "**Enter the correct path to the plate map file**. This will serve as the plate map for relating all subsequent information."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plate_map_fp = './test_data/Plate_Maps/2024_NPH_Plates_3-6_plate_map.tsv'\n",
+    "\n",
+    "if not os.path.isfile(plate_map_fp):\n",
+    "    print(\"Problem! %s is not a path to a valid file\" % file)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Read in the plate map**. It should look something like this:\n",
+    "\n",
+    "```\n",
+    "Sample\tRow\tCol\tBlank\n",
+    "GLY_01_012\tA\t1\tFalse\n",
+    "GLY_14_034\tB\t1\tFalse\n",
+    "GLY_11_007\tC\t1\tFalse\n",
+    "GLY_28_018\tD\t1\tFalse\n",
+    "GLY_25_003\tE\t1\tFalse\n",
+    "GLY_06_106\tF\t1\tFalse\n",
+    "GLY_07_011\tG\t1\tFalse\n",
+    "GLY_18_043\tH\t1\tFalse\n",
+    "GLY_28_004\tI\t1\tFalse\n",
+    "```\n",
+    "**Make sure there a no duplicate IDs**. If each sample doesn't have a different name, an **error** will be thrown and you won't be able to generate a sample sheet."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "##need to specify the directory for qiita.oauth.cfg\n",
```

Comment:
Does this comment mean the user is supposed to input something here?  If so, I think it needs more elaboration!  But if it is a left-over, let's take it out.

Reply:
it is somewhat of a vestige for a functionality that is not being used routinely. Maybe @charles-cowart or @antgonza can comment? I believe the qiita.oauth.cfg file was used for authenticating a user functionality that would allow the notebook to check sample names against the metadata associated with the study.

Reply:
@RodolfoSalido That is correct! It is for use with read_plate_map_csv() that will optionally connect to Qiita and compare sample-names in a project with what's in the plate-map file. It's still an option that's available in the sample-sheet creation notebook (and perhaps more?).